app/views/users/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<h2>Register</h2>
+
+<div class="row">
+  <div class="col-md-3">
+    <%= form_for @user do |f| %>
+
+      <div class="form-group">
+        <%= f.label :username %>
+        <%= f.text_field :username, class: "form-control" %>
+
+        <%= f.label :email %>
+        <%= f.text_field :email, class: "form-control" %>
+
+        <%= f.label :password %>
+        <%= f.text_field :password, class: "form-control" %>
+
+        <%= f.label :password_confirmation, "Password Confirmation" %>
```

Comment:
I'm being finicky and want "confirmation" to be capitalized. If this bothers you feel free to change it.